spec/models/performance_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe Performance, type: :model do
+  subject { build(:performance) }
+
+  it { is_expected.to validate_presence_of(:name) }
+  it { is_expected.to validate_presence_of(:date) }
+  it { is_expected.to validate_uniqueness_of(:date) }
+  it { is_expected.to validate_presence_of(:window_open) }
+  it { is_expected.to validate_uniqueness_of(:window_open) }
+  it { is_expected.to validate_presence_of(:window_close) }
+  it { is_expected.to validate_uniqueness_of(:window_close) }
+
+  describe 'validations' do
+    let(:performance) { build(:performance, window_open: window_open, window_close: window_close) }
+
+    context 'when window_open is earlier than window_close' do
+      let(:window_open) { Time.zone.now }
+      let(:window_close) { Time.zone.now + 1.day }
+
+      specify { expect(performance).to be_valid }
```

Comment:
Correct. `specify` is just a shorter syntax that you can use when an `it` block would be kinda useless.

In this case, the `it` block would just be something like:

```ruby
it 'is valid' do
  expect(performance).to be_valid
end
```

The 'is valid' is redundant when there is already the `be_valid` there

Some people prefer the `it`-only way, though, so I can change it if you'd like 👍